src/pp.js

```diff
@@ -131,16 +131,26 @@ function hasHardLine(doc) {
   });
 }
 
-function _makeIndent(n) {
+function _makeIndent(tabWidth) {
   var s = "";
 
-  for (var i = 0; i < n; i++) {
+  for (var i = 0; i < tabWidth; i++) {
     s += " ";
   }
 
   return s;
 }
 
+function _repeatIndent(n, indent) {
```

Comment:
By the way, `String.prototype.repeat` was added since node 3. This project doesn't support node 0.12 so you don't even need to implement this. You can use `indent.repeat(n)` instead :)